fenix/app/metrics.yaml

```diff
@@ -9210,6 +9210,23 @@ android_autofill:
     expires: 122
 
 home_menu:
+  help_tapped:
+    type: event
+    description: The user clicked the help button in home menu.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1850833
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-android/pull/3669
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+      - cgordon@mozilla.com
```

Comment:
The new process will not require a person's email.  Can you please remove the line with cgordon@mozilla.com.  Thanks